.github/CODEOWNERS

```diff
@@ -563,14 +563,13 @@
 
 # ServiceLabel: %Monitor %Service Attention
 /sdk/monitor/                                                      @SameergMS @dadunl
+/sdk/monitor/ci.yml                                                @nisha-bhatia @JoshLove-msft @pallavit @Azure/azure-sdk-write-monitor-data-plane
```

Comment:
Do we need the individual accounts if we're moving to a team contact?

Reply:
For now we definitely do -- the team PR auto-assign currently can only round-robin up to 7 members from a team, and for any team greater than 7 members this is problematic